src/drivers/imx/sai.c

```diff
@@ -242,7 +242,7 @@ static inline int sai_set_config(struct dai *dai,
 	dai_update_bits(dai, REG_SAI_XCR5(REG_RX_DIR), mask_cr5, val_cr5);
 	/* turn on (set to zero) stereo slot */
 	dai_update_bits(dai, REG_SAI_XMR(REG_RX_DIR), REG_SAI_XMR_MASK,
-			~(BIT(0) | BIT(1)));
+			(BIT(0) | BIT(1)));
```

Comment:
Need to check, it's possible the comment is wrong rather than the code. Give me a moment. @dbaluta any comment?

Reply:
I have checked the documentation and the code with ~ is in fact correct. Bits set to 0 are enabled and bits set to 1 are disabled, and XMR_MASK is MASK(31, 0). When turning off the channels you would write all ones in the register. Comment is just confusing so this change is incorrect.

Reply:
@paulstelian97 so we need to clarify the comment, the code is correct.

Reply:
This change wasn't intended, it's relic of previous build problem resolution in this line - fixed.